docs/integrations/tensorboard.rst

```diff
@@ -3,74 +3,427 @@
 
+Step 2: Convert your TensorBoard logs directory with Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**That's it! You can now browse and collaborate on your TensorBoard runs in Neptune.**
+Navigate to your TensorBoard logs directory and run:
 
-- Organize your TensorBoard experiments:
+.. code-block:: bash
 
-.. image:: ../_static/images/integrations/tensorboard_1.png
-   :target: ../_static/images/integrations/tensorboard_1.png
-   :alt: organize TensorBoard experiments in Neptune
+    neptune tensorboard --project USER_NAME/PROJECT_NAME
 
-- Compare your TensorBoard runs:
+Alternatively, you can point Neptune to your TensorBoard logs directory:
 
-.. image:: ../_static/images/integrations/tensorboard_2.png
-   :target: ../_static/images/integrations/tensorboard_2.png
-   :alt: compare TensorBoard runs in Neptune
+.. code-block:: bash
 
-- Share your work with others by sending an |experiment-link|.
+    neptune tensorboard /PATH/TO/TensorBoard_logdir --project USER_NAME/PROJECT_NAME
 
-.. External links
+.. tip::
 
-.. |GitHub| raw:: html
+    You can also set the project qualified name in the ``NEPTUNE_PROJECT`` environment variable and skip the ``--project`` argument
 
-    <a href="https://github.com/neptune-ai/neptune-tensorboard" target="_blank">GitHub</a>
+	Linux/macOS:
 
-.. |neptune-tensorboard| raw:: html
+	.. code:: bash
 
-    <a href="https://github.com/neptune-ai/neptune-tensorboard" target="_blank">neptune-tensorboard</a>
+		export NEPTUNE_PROJECT='USER_NAME/PROJECT_NAME'
+
+	Windows:
+
+	.. code-block:: bat
+
+		set NEPTUNE_PROJECT="USER_NAME/PROJECT_NAME"
+
+Browse and collaborate on your TensorBoard runs in Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can now click on one of the links from the console and go to Neptune to see your experiments.
+
+**Dashboard**
+
+.. image:: ../_static/images/integrations/tensorboard_dashboard.png
+   :target: ../_static/images/integrations/tensorboard_dashboard.png
+   :alt: Organize TensorBoard experiments in Neptune
+
+|example-dashboard|
+
+**Comparison**
+
+.. image:: ../_static/images/integrations/tensorboard_compare.png
+   :target: ../_static/images/integrations/tensorboard_compare.png
+   :alt: Compare TensorBoard experiments in Neptune
+
+|example-compare|
+
+Remember that you can try it out with zero setup:
+
+|tensorboard-code-example|
+
+.. _tensorboard-log-to-neptune-via-tensorboard:
+
+Log new runs to Neptune via TensorBoard
+---------------------------------------
+
+Before you start
+^^^^^^^^^^^^^^^^
+#. Ensure that you have ``Python 3.x`` and following libraries installed:
+
+   * ``neptune-tensorboard``
+   * ``tensorflow``
+   * ``tensorboard``. See the |tensorboard-install|.
+
+.. code-block:: bash
+
+  pip install --quiet tensorflow tensorboard neptune-tensorboard
+
+#. You also need minimal familiarity with TensorBoard. Have a look at the |tensorboard-install| guide to get started.
```

Comment:
drop "guide" as it makes repetition.

Reply:
It look like this in the docs: `Have a look at the TensorBoard get started guide guide to get started.`